kusto/kusto.go

```diff
@@ -431,3 +434,19 @@ func (*Client) contextSetup(ctx context.Context, mgmtCall bool) (context.Context
 func (c *Client) HttpClient() *http.Client {
 	return c.http
 }
+
+func (c *Client) Close() error {
+	var err error
+	if c.conn != nil {
+		err = c.conn.Close()
+	}
+	if c.ingestConn != nil {
+		err2 := c.ingestConn.Close()
+		if err == nil {
+			err = err2
+		} else {
+			err = fmt.Errorf("combined error: %v %v", err, err2)
```

Comment:
Same

Reply:
Also, note if you nest these you'll have multiple "combined error:" prefixes

Reply:
Fixed